Sources/Rendering/Core/ImageSlice/index.d.ts

```diff
@@ -111,6 +117,23 @@ export interface vtkImageSlice extends vtkProp3D {
 	 * and the Renderer will do the images in their own pass.
 	 */
 	hasTranslucentPolygonalGeometry(): boolean;
+
+	/**
+	 * 
+	 */
+	makeProperty(): vtkImageProperty;
```

Comment:
`makeProperty()` has the same signature as `vtkImageProperty.newInstance()`

Reply:
ah, so it needs to take optionally IImageSliceInitialValues right?